mypy/stubtest.py

```diff
@@ -1382,6 +1387,8 @@ def anytype() -> mypy.types.AnyType:
     elif isinstance(runtime, (bool, int, str)):
         value = runtime
     else:
+        if isinstance(runtime, type):
+            return mypy.types.TypeType(fallback)
```

Comment:
I decided to use nested `if` because all other `elif` cases work with literal values. I didn't want to created mixed concepts here.

And since we only work with `fallback` under `else` - I kept it that way.

Reply:
I see -- maybe we should move this block to just below the `if isinstance(runtime, tuple)` block in that case? That seems like a more logical place for it to be, and it would mean we could have it as an isolated block of code instead of next to all the literal-value stuff.